packages/p2p/src/components/buy-sell/__test__/buy-sell-table.spec.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { isDesktop } from '@deriv/shared';
+import { useStores } from 'Stores';
+import BuySellTable from '../buy-sell-table.jsx';
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(),
+}));
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: () => <div>Loading</div>,
+}));
+
+jest.mock('Components/table/table-error.jsx', () => ({
+    ...jest.requireActual('Components/table/table-error.jsx'),
+    TableError: ({ message }) => <div>{message}</div>,
+}));
+
+jest.mock('@deriv/shared/src/utils/screen/responsive', () => ({
+    ...jest.requireActual('@deriv/shared/src/utils/screen/responsive'),
+    isDesktop: jest.fn(),
```

Comment:
@artsemkavaleu-deriv believe that we can mock return value right here:
isDesktop: jest.fn(()=>true) or isDesktop: jest.fn().mockReturnValue(true)
Therefore we can remove 3 and 81 strings

Reply:
right, done, thx for notice!